### DISCUSSION TITLE
Storage overview by file type

### DISCUSSION THREAD

**Discussion by Wjxfi** [d4rken-org/sdmaid-se#354] [Ideas]
In addition to the usual file manager look, it also looks interesting

When you click on them, it will show you the similar files. This is convenient

![aadfbec7-9e38-4de6-8ae3-9f8e6a2ea424.jpg](https://user-images.githubusercontent.com/113500363/235653742-aa51a833-2212-44ef-9aff-f21a41f582fc.jpg)

https://www.f-droid.org/packages/com.simplemobiletools.filemanager.pro/ did it

Do as he did

'

I think this should be added because your application can find more hidden files and point them to partitions. Or create new partitions.'

the files in the sections can be sorted by:

name,

size,

time of change,

extension

----

ascending,

descending

all asked as in this app
https://www.f-droid.org/packages/com.simplemobiletools.filemanager.pro/

**Comment by d4rken** [comment]
This would be an alternative view for the StorageAnalyzer tool?

I need to see the biggest files, but how does sorting them by type help? :thinking: 

Due to how storage access works, it will also never be possible to show accurate size numbers for the different types (except on rooted devices).

**Comment by Wjxfi** [comment]
> This would be an alternative view for the StorageAnalyzer tool?

Yes, a more convenient alternative


> Due to how storage access works, it will also never be possible to show accurate size numbers for the different types (except on rooted devices).
> 

It is necessary to show the approximate numbers. This is very helpful. and when root rights are available this can be shown.
